### PR TITLE
[2.10] Update Exposing Dashboard doc in light of Viz

### PR DESCRIPTION
**Pull request by alpeb**

### CODE REVIEW

File: linkerd.io/content/2/tasks/exposing-dashboard.md [hunk 1]
```diff
@@ -211,9 +211,12 @@ spec:
...
         - name: web
           args:
-            - -api-addr=linkerd-controller-api.linkerd.svc.cluster.local:8085
-            - -grafana-addr=linkerd-grafana.linkerd.svc.cluster.local:3000
+            - -linkerd-controller-api-addr=linkerd-controller-api.linkerd.svc.cluster.local:8085
+            - -linkerd-metrics-api-addr=metrics-api.linkerd-viz.svc.cluster.local:8085
+            - -cluster-domain=cluster.local
+            - -grafana-addr=grafana.linkerd-viz.svc.cluster.local:3000
             - -controller-namespace=linkerd
+            - -viz-namespace=linkerd-viz
```
Comment (Pothulapati) [inline linkerd.io/content/2/tasks/exposing-dashboard.md:219]: Can you also update the namespace in line 189?

**Review by Pothulapati — APPROVED** [review]
LGTM

File: linkerd.io/content/2/tasks/exposing-dashboard.md (hunk 1 above)
Comment (alpeb) [reply linkerd.io/content/2/tasks/exposing-dashboard.md:220]: That line didn't point to any particular namespace from what I could see, but I've just pushed more changes, so please give it another look.

Comment (Pothulapati) [reply linkerd.io/content/2/tasks/exposing-dashboard.md:220]: LGTM

**Review by kleimkuhler — APPROVED** [review]

**Pull request merged**